integrationtests/test_all_examples.py

```diff
@@ -279,6 +281,24 @@ def run_file(file_path):
         exec(content, globals)
 
 
+def load_demo(file_path, variable_name="demo"):
+    """ Loads a demo example from given file_path. Extracts the relevant
+    object via variable_name.
+    Parameters
```

Comment:
There is a missing newline before `Parameters`.  It is missing in the `run_file` function too. Feel free to add that too as a drive-by (small enough to not worth a separate PR).